dissect/target/plugin.py

```diff
@@ -540,74 +531,160 @@ def decorator(obj):
     return decorator
 
 
-def plugins(osfilter: Optional[type[OSPlugin]] = None) -> Iterator[PluginDescriptor]:
-    """Retrieve all plugin descriptors.
+def plugins(
```

Comment:
`target-query ~/TestSystemen/SCHARDT.img -n -q -f os.windows._os`

has now different behaviour (compared to main).